exercises/resistor-color/resistor-color.example.ts

```diff
@@ -0,0 +1,3 @@
+export const COLORS = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
+
+export const colorCode = (color: string): number => COLORS.indexOf(color)
```

Comment:
Since its typescript, you can restrict the function to only allow specific strings by creating a `type`
I.e.
```
 type Colors = "black" | "brown"......
export const colorCode = (color: Colors) =>....
```

Reply:
It's only an example, not a canonical optimal solution. That said, I definitely agree with only allowing certain strings.